### ISSUE TITLE
Greater Blessings not giving paladins any threat

### ISSUE THREAD

**Issue by HunLepto** [the-hyjal-project/bugtracker#180]
**Describe Your Issue**

Greater blessings should generate aggro based on several factors, but it doesn't do anything on the server.
**Steps To Reproduce**

1. Go in combat, have someone hit a mob and use greater blessing on the friendly target to see if you get any threat from using a greater blessing on him/her.
2. Go in combat, have several people of the same class hit a mob and use greater blessing on one of the friendly targets to see if you get any threat from using a greater blessing on him/her.
3. 

Item: https://classic.wowhead.com/item=
Quest: https://classic.wowhead.com/quest=
Spell: https://classic.wowhead.com/spell=25898

**Expected Behavior**

It should generate threat based on several factors: Base Threat + Mana Cost x Number of Players Buffed.

**Comment by HunLepto** [comment]
Forgot to write, Greater Blessing of Wisdom is an exception to this as it has no base threat.

**Comment by Nolin7777** [comment]
Where is your source that Greater Blessing of Wisdom has an exception and should generate threat based on several factors: Base Threat + Mana Cost x Number of Players buffed?

**Comment by HunLepto** [comment]
I may have misspelled it, Greater Blessing of Wisdom is an exception AS IN
not generating any threat, because it has no Base Threat basically. I
apologize for that.

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Mentes
a vírusoktól. www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

놀린 <notifications@github.com> ezt írta (időpont: 2021. jan. 7., Cs, 18:39):

**Comment by theloras** [comment]
additional information for reference:

for what it's worth guys - regarding GBoK spam -  i went straight to the source and asked Kevin Jordan himself about it and RF
Theloras09/25/2019
@Kevin Jordan - is this working correctly/as intended from your pov?
https://www.youtube.com/watch?v=6Qv-Povncwo

TLDR - no one ever thought that spamming Blessings and Seals could produce agro - all of which are modified by Righteous Fury
who needs a taunt lol

Kevin Jordan09/25/2019
Buffs do generate threat and Greater Blessings buff a number of people.  I don't remember the values but they should not generate any more or less threat than any other buff.  What matters most here is the number of targets (running a lot of mages are you? :P) and Righteous Fury modifying it.

Theloras09/25/2019
was RF supposed to boost threat from everything and not just holy dmg abilities?
cuz that was my and every other theory crafter's thought for years
and RF boosting Seals/Blessings made us all /boggle come classic

Kevin Jordan09/25/2019
Holy attacks only.

Theloras09/25/2019
ok so then this is a bug?

Kevin Jordan09/25/2019
if it's modifying greater blessing threat generation, yes.

Theloras09/25/2019
ok thx for the feedback

so the TLDR is - Kevin copy pasted the code from defensive stance to boost threat
and forgot to limit RF to damage spells only - since this code was never changed
#NotABug on Classic
this was the only net gain Paladins got in Classic
https://classic.wowhead.com/news=295877/oct-28th-update-to-not-a-bug-list-thrash-mechanic-righteous-fury-sharpening-ston

https://media.discordapp.net/attachments/530489505057341440/640307487492276224/Greater_Blessing_of_Kings_Threat_Generation.png

https://cdn.discordapp.com/attachments/530489505057341440/775437512579219466/unknown.png
